### DISCUSSION TITLE
I'm new here and i'm lokking for some help with data management

### DISCUSSION THREAD

**Discussion by meeshiura** [community/community#61365] [Other Feature Feedback, Questions, & Ideas]
Hi
I am new here so I'm not sure where to start so I decided to start looking for help here. The thing is that I have a problem where my stats between Adform RTB platform and new GA4 disappear. Differences (depending on the day) oscillate in between the range of 40-50%. I'm trying to locate the source of the problem and I wanted to ask if any of you had a similar problem maybe and if you managed to solve it somehow. Thanks. :)

**Comment by queenofcorgis** [comment]
Hi @meeshiura,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on the tools you mentioned. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
